src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -26,15 +25,17 @@ const GuideStore = Reflux.createStore({
       org: null,
       // Current project.
       project: null,
-      // Total events received in the project in the last 30 days. id (int) -> int.
-      projectStats: {},
-      // Whether the project has customized alert rules. id (int) -> bool.
-      projectRules: {},
       // We force show a guide if the URL contains #assistant.
       forceShow: false,
       // The previously shown guide.
       prevGuide: null,
     };
+
+    // Total events received in the project in the last 30 days. id (int) -> int.
+    this.projectStats = new Map();
```

Comment:
For some reason the `projectId` was being saved into the object with a `string` as key instead of an `int` (even though we coerce it into an `int`).

Also moving this out of `state` since it doesn't look like any components need this exposed, but can be moved back into `state` if necessary.